commit 20114e0e8278ddc3dc5544a529198fa8ec107ac1
Author: Sage Weil <sweil@redhat.com>
Date:   Tue Mar 7 16:28:24 2017 -0500

    mon/MonCap: pass entity type (mgr, mon) to is_capable
    
    This lets us define different meanings for profiles on
    the mon and mgr.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index b154b8eee2b..b6e2bc6079b 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -351,11 +351,15 @@ bool DaemonServer::_allowed_command(
   bool cmd_w = this_cmd->requires_perm('w');
   bool cmd_x = this_cmd->requires_perm('x');
 
-  bool capable = s->caps.is_capable(g_ceph_context, s->entity_name,
-                                    module, prefix, param_str_map,
-                                    cmd_r, cmd_w, cmd_x);
-
-  dout(10) << " " << (capable ? "" : "not ") << "capable" << dendl;
+  bool capable = s->caps.is_capable(
+    g_ceph_context,
+    CEPH_ENTITY_TYPE_MGR,
+    s->entity_name,
+    module, prefix, param_str_map,
+    cmd_r, cmd_w, cmd_x);
+
+  dout(10) << " " << s->entity_name << " "
+	   << (capable ? "" : "not ") << "capable" << dendl;
   return capable;
 }
 
